### PR TITLE
Cortx-multisite: resolving codacy errors

### PR DESCRIPTION
**Pull request by Mayank-Sriva**
# Problem Statement
- Fix codacy issues for Cortx-multisite repo

## Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

## Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

## Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

## Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

## Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

## Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: test/s3_test_script.py [hunk 1]
```diff
@@ -121,7 +120,7 @@ def create_iam_role(role_name):
 
     session = boto3.session.Session(profile_name='default')
     iam = session.client('iam')
-    response = iam.create_role(
```
Comment (cdeshmukh) [inline test/s3_test_script.py:124]: its always good to collect response, why removing? was it throwing warning that response var was unused. if yes, just print respone[status] like in other functions

File: test/s3_test_script.py [hunk 2]
```diff
@@ -235,7 +235,9 @@ def verbose(verbose, prompt, message):
     if not verbose:
         return
 
-    create_iam_role(src_bucket, dest_bucket)
+    # Commenting out due to codecy issue
+    # Commenting out doesn't seem to make any impact on code flow
```
Comment (cdeshmukh) [inline test/s3_test_script.py:239]: replace above comments (line 238-239) with 
# Call here taking two arguments even if function definition has one, and codacy flagged it.
# Commenting out given no functional need seen and no impact seen on removal

**Review by cdeshmukh — COMMENTED** [review]
very minimal codacy issues fixed till now

File: test/s3_test_script.py [hunk 3]
```diff
@@ -235,7 +236,9 @@ def verbose(verbose, prompt, message):
     if not verbose:
         return
 
-    create_iam_role(src_bucket, dest_bucket)
+    # Call here taking two arguments even if function definition has one, and codacy flagged it.
+    # Commenting out given no functional need seen and no impact seen on removal
+    # create_iam_role(src_bucket, dest_bucket)
```
Comment (mbcortx) [inline test/s3_test_script.py:241]: ok

**Review by mbcortx — APPROVED** [review]
+1

**Review by cdeshmukh — APPROVED** [review]

**Review by priyanka25081999 — APPROVED** [review]

**Pull request merged**